Fabulous.XamarinForms/src/Fabulous.XamarinForms/Xamarin.Forms.Core.json

```diff
@@ -2822,6 +2791,137 @@
       "primaryConstructorMembers": [
         "Items"
       ]
+    },
+    {
+      "type": "Xamarin.Forms.IndicatorView",
+      "name": "Indicator",
```

Comment:
Is it wanted you renamed it to `Indicator`?
I think it's safe to just remove the `name` attribute from the JSON.